__tests__/app.test.js

```diff
@@ -81,3 +81,62 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("status:200 should update the votes property of an article by increasing the votes", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 1 })
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Object);
+        expect(body.article_id).toBe(1);
+        expect(body.votes).toBe(101);
+      });
+  });
+  test("status:200 should respond with the updated article", () => {
```

Comment:
This test may fail as you move on. Consider checking that the updated article 'contains' the correct keys ;) this gives you more room to add keys in the future.